### DISCUSSION TITLE
Need help with the Read me

### DISCUSSION THREAD

**Discussion by doobybooby** [community/community#23764] [New to GitHub]
<p>So I’m new to Github and learning how to post my code on Github. I know the importance of the Readme files but I’m struggling with the headers.</p>
<p>I understand there is</p>
<h1>Header1</h1>
<h2>Header2</h2>
<h6>Header3</h6>
<p>but when I type in the</p>
<h1>Example</h1>
<p>it will literally print the “hastag Example”<br>
Other variations didn’t help</p>
<p><span class="hashtag">#Example</span>, returned “<span class="hashtag">#Example</span>”</p>
<p>Am I missing something?</p>
<p>After I posted this I realized the headers are working properly in this github community but not in my Readme in GitHub…</p>

**Comment by logankilpatrick** [comment]
<p>Hey! Can you link a specific readme?</p>
<p>Note that you need to add a space after the “#” which may be your problem.</p>

**Comment by doobybooby** [comment]
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/doobybooby/TicTakToe/blob/f37fd2b34691b1d92d633bd79223e36847bf8dad/getting_started" target="_blank" rel="nofollow noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/doobybooby/TicTakToe/blob/f37fd2b34691b1d92d633bd79223e36847bf8dad/getting_started" target="_blank" rel="nofollow noopener">doobybooby/TicTakToe/blob/f37fd2b34691b1d92d633bd79223e36847bf8dad/getting_started</a></h4>
<pre><code class="lang-"># Getting Started

&gt; To create TicTakToe you need functions to display the board, take in a user input and change the board accordingly
&gt; Here are some functions you are going to have to understand in order to build the game
&gt; It is easier if you divide the problem into smaller sub-problems

* Display the board - displayBoard()

* Ask user for the position of the board they want to change - positionChoice()

* Replace the old choice with the new user choice - replacementChoice()

* Ask user if they wish to continue playing - keepPlaying()

</code></pre>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p><a class="mention" href="/logankilpatrick">@logankilpatrick</a><br>
Here is the link to my repository I’m having an issue with. as you can see it is not indenting, nor doing the bullet points</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Try naming your file with a .md ending, my guess is that without that it’s not recognized as markdown, and thus not rendered as such. Also, if you want it to be recognized as the README for your repository it’d help if you named it that. :wink:</p>

**Comment by doobybooby** [comment]
<p>Thank you, I was struggling with another private read me page, but thought this link I shared was more appropriate and contained more examples. ending the file with .md fixed everything. Thank you</p>
